fusor/models.py

```diff
@@ -54,11 +55,11 @@ class FunctionalDomain(BaseModel):
     """Define FunctionalDomain class"""
 
     type: Literal[FUSORTypes.FUNCTIONAL_DOMAIN] = FUSORTypes.FUNCTIONAL_DOMAIN
-    id: CURIE
-    name: StrictStr
     status: DomainStatus
-    gene_descriptor: GeneDescriptor
-    location_descriptor: LocationDescriptor
+    associated_gene: GeneDescriptor
+    id: Optional[CURIE]
```

Comment:
Not a big deal, but wondering if we should do `_id` since it's now optional? To follow VRS conventions